pinc/links.inc

```diff
@@ -78,3 +78,46 @@ function private_message_link($proofer_username, $target = "_blank")
 
     return $a;
 }
+
+// -----------------------------------------------------------------------------
+
+function return_to_project_page_link($projectid, $url_suffix = "")
```

Comment:
we should mention that the url_suffix needs to be url encoded since we don't encode before we craft the HTML. That or we should encode and make sure the callers don't encode.